### PR TITLE
Recursion length of a string 

### LINKED ISSUE #14: Recursion: Length of a String
Write a function that returns the length of a string. Make your function recursive.

Examples
```
length("apple") ➞ 5

length("make") ➞ 4

length("a") ➞ 1

length("") ➞ 0
```

Notes
Check the Resources tab for info on recursion.

### PR DESCRIPTION
**Pull request by Anishluke92**

### CODE REVIEW

File: RecursionLengthOfAString.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+length("a") ➞ 1
+
+length("") ➞ 0
+Notes
+Check the Resources tab for info on recursion.
+
+=end
+
+def length(string)
```
Comment (danielpaul) [inline RecursionLengthOfAString.rb]: not recursive. the important thing here is to write a recursive method that would do this.

**Review by danielpaul — CHANGES_REQUESTED** [review]

**Comment by danielpaul** [comment]
@Anishluke92 there are two or 3 other questions in this PR also? have a look at the file changes

File: RecursionLengthOfAString.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  if string == '' 
+    return 0
+  else 
+    return 1 + length(string.chop)  
+  end
+  
+end
```
Comment (danielpaul) [inline RecursionLengthOfAString.rb]:
```suggestion
def length(string)
  return 0 if string == '' 
  1 + length(string.chop)  
end
```

**Review by danielpaul — CHANGES_REQUESTED** [review]

**Comment by danielpaul** [comment]
This is ready but you have a merge conflict. So task: Learn about merge conflicts and learn how to resolve them. once you know how, you can resolve this merge conflict and this can be merged in then.

**Comment by Anishluke92** [comment]
Merge conflict ? whats does that do?

**Pull request merged**